src/common/thirdparty/clone-catapult.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+git clone https://github.com/catapult-project/catapult.git $HOME/catapult
+pushd $HOME/catapult
```

Comment:
We shouldn't mess with users home directories. If we clone it, it should be in some Ray specific location.

We also need to make sure that it gets copied to the right location if we install it with `python setup.py install` or `pip install`.